icu4j/main/tests/framework/src/com/ibm/icu/dev/test/UnicodeKnownIssues.java

```diff
@@ -0,0 +1,151 @@
+package com.ibm.icu.dev.test;
```

Comment:
The failing copyright checker build bot reminds us that the copyright lines are missing. Please add.
```
// © 2021 and later: Unicode, Inc. and others.
// License & terms of use: https://www.unicode.org/copyright.html
```
(Hopefully the checker accepts https. I just snuck that in there.)